Frameworks/UIKit/UIColor.mm

```diff
@@ -715,7 +710,7 @@ - (BOOL)getHue:(float*)h saturation:(float*)s brightness:(float*)v alpha:(float*
         *v = (float)out.v;
     }
     if (a) {
-        *a = _a;
+        *a = _components.a;
     }
 
     return TRUE;
```

Comment:
I know it was already here, but for consistency could we replace shouty defines with non-shouty ones?